jrportfolio.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Jason Rees Portfolio</title>	
+		<link href="normalize.css" rel="stylesheet" type="text/css">
+		<link href="jrportfolio.css" rel="stylesheet" type="text/css">
+</head>
```

Comment:
missing the responsive meta tag from the head

Reply:
I have moved the link tags and added the responsive meta tag